README.md

```diff
@@ -21,26 +23,26 @@ tar -zxvf cf.tgz
 sudo mv cf /usr/local/bin
 ```
 
-Confirm the installation:
+### Confirm the installation
 
 ```bash
 cf --version
 ```
 
 As of this writing the current cf CLI version is `6.10.0-b78bf10`.
-	
+
 ## Setting up your account
 
-You will need a Cloud Foundry account before continuing. Please create an issue in the [DevOps issue tracker](https://github.com/18F/DevOps/issues) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
+You will need a Cloud Foundry account before continuing. Please [create an issue in the DevOps issue tracker](https://github.com/18F/DevOps/issues/new) and assign it to [@dlapiduz](https://github.com/dlapiduz) or [@ozzyjohnson](https://github.com/ozzyjohnson).
 
-To login, run:
+### Log in
 
 ```bash
 cf api --skip-ssl-validation https://api.cf.18f.us
```

Comment:
Have we figured out what our final domain will be for the PaaS? Something at 18f.gov?